scripts/bfasst/compare/waveform.py

```diff
@@ -144,12 +156,12 @@ def check_compare_status(self, log_path, default=CompareStatus.NOT_EQUIVALENT):
             return Status(CompareStatus.TIMEOUT)
 
         # Regex search for result
-        m = re.search(r"Equivalence successfully proven!", log_text, re.M)
-        if m:
+        i = re.search(r"Equivalence successfully proven!", log_text, re.M)
```

Comment:
This doesn't need to be a raw string.  More readable to just have it be a normal string.

Reply:
This check_compare_status function is from whoever made conformal.py, onespin.py, and yosys.py. I'll make sure to fix it, but I suspect these changes will have to be done on the rest of the python scripts in compare as well.